nss-cohort-36/bangazon-client-virtual-magicians-1
# Cb models

# Description
Just adding APIManager.js. Not sure this is really worth a PR but here we are.

Please delete options that are not relevant.

# Testing Instructions
Make sure to commit local changes in your branch before switching to master.
1. `git checkout master`
2. `git fetch --all`
3. `git checkout cb-models`
4. Added modules directory and APIManager.js so just confirm they exist?
# Checklist:
